terraform/aws/post-config-k8s/iam.tf

```diff
@@ -185,4 +185,39 @@ resource "aws_s3_bucket" "backup_bucket" {
   bucket        = var.backup_bucket_name
   force_destroy = var.backup_bucket_force_destroy
   tags          = merge({ Name = var.backup_bucket_name }, var.tags)
+}
+
+# EBS CSI driver
+resource "aws_iam_role" "csi_role" {
+  name  = "${local.base_domain}-csi"
+  count = var.create_csi_role ? 1 : 0
+  assume_role_policy = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Action": "sts:AssumeRole",
+      "Principal": {
+        "Service": "ec2.amazonaws.com"
+      },
+      "Effect": "Allow",
+      "Sid": ""
+    }
+  ]
+}
+EOF
+}
```

Comment:
add tags on the role:   tags               = merge({ Name = "${local.base_domain}-csi-role" }, var.tags)

Reply:
Noted will action